Assets/Scripts/Character.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+public class Character : MonoBehaviour
+{
+    public float moveSpeed = 5f;
+    public int maxHealth = 100;
+    public int attackDamage = 10;
+    public int defense = 10;
+    public int dexterity = 5;
+    public string weapon; //we can later create a weapon class and use that instead of string
+    protected int currentHealth;
+    protected Rigidbody2D rb;
```

Comment:
The enemies might not need a RigidBody just yet. We should be able to handle enemies with just a collider, which could make things simpler for us down the line. More important though, RigidBody has a non-insignificant performance cost since it essentially makes the gameobject an interactable part of the physics systems. This won't scale well with a lot of enemies.

I recommend making the `Move` method abstract, carrying the insides of the `Move` method to the player object, and overriding the enemy movement on its own class with the `Transform` component instead of `RigidBody`. 

Example for using `Transform`:

```
        Vector2 position = transform.position;
        position.x -= (speedActual * Time.deltaTime);
        transform.position = position;
```

Reply:
kardeşim non-insignificant ne, dümdüz significant yazsana; Anti-antifa gibi

Reply:
We should still keep the RigidBody in the `PlayerController`. Currently, since it's removed from `Character`, it's not inherited anymore.